spec/fixtures/notify/2b9f7f53-1f28-446d-9502-cb77d0928f61.yml

```diff
@@ -0,0 +1,13 @@
+id: "2b9f7f53-1f28-446d-9502-cb77d0928f61"
+name: "email_confirmation_for_creator_en"
+subject: "Please confirm your email"
+body: |-
+  Click this link to confirm your email:
```

Comment:
Probably add a comment about moderation here that you can crib from the other templates. I suppose it should also have Christmas and Easter variants too but maybe for another story.